Gems/Atom/Tools/AtomToolsFramework/Code/Include/AtomToolsFramework/Document/AtomToolsDocumentMainWindow.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
```

Comment:
Is still still required? Would you happen to know what library Qt is conflicting with?

Reply:
This causes Qt to ignore lines or files when it's processing for code generation. This file started as a copy of the material editor main window and was stripped down from there. I did not add the line as part of the revision or bothered to test removing it since it only affects code generation.